linux/src/async_socket_linux.c

```diff
@@ -50,6 +54,7 @@ MU_DEFINE_ENUM_STRINGS(ASYNC_SOCKET_SEND_SYNC_RESULT, ASYNC_SOCKET_SEND_SYNC_RES
 #define THREAD_COUNT    1
 
 static volatile_atomic int32_t g_thread_access_cnt;
```

Comment:
>g_thread_access_cnt [](http://example.com/codeflow?start=31&length=19)

I find this VERY weird that it is not initialized to "anything".

I find it with interlocked_increment and interlocked_decrement in the file all over the place, but what's its initial value? 

And by the way, HOW did this pass valgrind ? I might be wrong and it COULD be initialized some place, just that I cannot find where that place is.